zbsprimarystorage/zbsprimarystorage/zdaemon.py

```
@@ -0,0 +1,53 @@
+import zbsagent
+
+
+def prepare_pid_dir(path):
+    pdir = os.path.dirname(path)
+    if not os.path.isdir(pdir):
+        os.makedirs(pdir)
+
+
+def main():
+    usage = 'usage: python -c "from zbsprimarystorage import zdaemon; zdaemon.main()" start|stop|restart'
+    if len(sys.argv) != 2 or not sys.argv[1] in ['start', 'stop', 'restart']:
```

Ruff (0.8.2)

Test for membership should be `not in`

Convert to `not in`

(E713)